python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -114,7 +93,7 @@ def __copy__(self, memo=None):
     def __deepcopy__(self, memo=None):
         cls = self.__class__
         obj = cls()
-        this_dict = self.__getstate__()
+        this_dict = self.__dict__
```

Comment:
I realise you haven't changed existing behaviour here but should this be `this_dict = copy.copy(self.__dict__)`? Otherwise the following line are altering the `__dict__` of *this* object.